### ISSUE TITLE
tensorflow custom op on Windows

### ISSUE THREAD

**Issue by colorfulCloud** [BowieHsu/Tensorflow-windows-dll#3]
have you ever tried to add custom op? Is it possible to load a compiled custom op dll and use tf.load_op_library(xxx.dll)

**Comment by BowieHsu** [comment]
@colorfulCloud yep, but in my way, I compiled "my_op.cc" into "my_op.lib" and then compiled "my_op.lib" into tensorflow.lib , then i try to run graph include "my_op" , it seems like everything works fine
